### PR TITLE
Added Auto, Kill, Stun

### PR DESCRIPTION
**Pull request by NoSloppy**
for blaster.h prop mod #176

### CODE REVIEW

File: sound/voice_data.h [hunk 1]
```diff
@@ -35,7 +34,11 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
...
...x48,0x82,0x3d,0x83,0x1a,0xdb,0xe6,0x90,0xf1,0x94,0xae,0x24,0x9b,0xc3,0xab,0x29,0xde,0x90,0xe2,0x8e,0xc2,0x2d,0xf4,0x4a,0x72,0x0a,0x92,0xb0,0x8c,0x2b,0xc9,0x2d,0x48,0x5c,0x2b,0x3f,0xa8,0x94,0x20,0x70,0xeb,0xbc,0x50,0x9a,0x82,0xc0,0xbd,0x6a,0x42,0x6a,0x8a,0xa7,0x0e,0x61,0x8f,0x38,0x06,0x58,0x7e,0x42,0x01,0xd3,0x75,0x10,0xbf,0x5b,0x63,0x29,0x4d,0x63,0xbc,0xb6,0xd2,0x2d,0xd5,0x4e,0x7a,0xda,0x2a,0xb7,0xd4,0x38,0xe1,0x6d,0x31,0x55,0xd3,0x9a,0x04,0xa5,0xd6,0x52,0x35,0x4f,0x92,0xe4,0x54,0x28,0xd3,0xd3,0x4c,0x91,0x73,0xc3,0x32,0x2f,0xa9,0x45,0x49,0x95,0xc7,0x35,0xa5,0x14,0x25,0x25,0x1a,0xab,0xa2,0xdb,0xf4,0x14,0x69,0x3c,0x5b,0x76,0x33,0x92,0xb7,0x6c,0x0b,0xca,0xc3,0x2c,0x22,0x35,0xdb,0x25,0x37,0x2b,0xf3,0xb0,0x18,0xb7,0x65,0x9c,0x28,0x5d,0xaa,0xcd,0xb6,0xf2,0x23,0x2f,0xad,0xf2,0xd8,0x2a,0x16,0xad,0x2d,0xd2,0x22,0xb9,0x48,0xc5,0x55,0x1f,0xb7,0x55,0xe2,0x4c,0x33,0x65,0x52,0xc9,0x88,0xbb,0x30,0xe5,0x0e,0xd9,0x23,0x69,0x42,0xd4,0x3b,0x2d,0xb7,0xb4,0x72,0x8a,0x19,0x93,0x9c,0xb2,0xc2,0xa8,0xae,0x85,0x4e,0xc8,0x0b,0xa3,0xbe,0x62,0xca,0x2e,0xcf,0x94,0x67,0x93,0x25,0x99,0x3c,0x51,0x9e,0x4d,0x96,0x64,0xf2,0xc2,0xb0,0x2f,0x45,0xb2,0xc8,0x33,0xc5,0xbe,0x40,0xc9,0x24,0x8f,0x84,0x6a,0x43,0x2c,0xc3,0x03};
```
Comment (profezzorn) [inline sound/voice_data.h:41]: The description doesn't mention any changes to spBATTLEVELCURR, so why was it changed?

Comment (NoSloppy) [reply sound/voice_data.h]: Updated. Was a placeholder copy/paste. Was going to add male version. Aborted, deleted.

Comment (NoSloppy) [reply sound/voice_data.h]: Same for BATTERYLVL removal, it was just unused. I suppose the Female BATTLVLCURR could also be removed, but removing unused is unnecessary anyway.

File: sound/voice_data.h [hunk 2]
```diff
@@ -35,7 +34,10 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
...
-#else
+const uint8_t spAUTO[] PROGMEM = {0xac,0x4c,0x2d,0x8c,0x57,0x9d,0xb8,0x32,0xed,0x34,0x7e,0x8a,0x53,0xca,0x78,0xc2,0x75,0xb5,0x4a,0x51,0xe2,0x8d,0xd0,0xb5,0xb2,0x43,0x49,0xbd,0x4a,0x3a,0x6a,0x2f,0x25,0xd5,0x4c,0xcd,0x8a,0x33,0x94,0x9c,0xbc,0xa4,0x63,0xac,0x33,0x72,0xd2,0xe2,0xf6,0x32,0xc6,0xca,0x96,0x4a,0xc6,0x2a,0x17,0xa7,0x04,0x0e,0xde,0x2c,0xdb,0xbc,0x12,0xa4,0x78,0xb3,0x6c,0x0b,0x4a,0x90,0xe2,0x8f,0x72,0x23,0x2a,0x4e,0x5b,0xa7,0x63,0x8f,0x38,0x3b,0x1d,0xad,0xaa,0x5d,0xe2,0xec,0x75,0x68,0xb3,0x6a,0x88,0x63,0xb2,0xc6,0xf7,0xb2,0x21,0x0e,0xc5,0x13,0x37,0xca,0xaa,0xd8,0x0e,0x73,0x7a,0x0f,0x23,0x62,0xbd,0xd5,0xe8,0x2c,0x16,0x89,0xf5,0x16,0xe5,0xf1,0x84,0x28,0xe6,0xcd,0x15,0x47,0x55,0xa3,0x98,0xc5,0x30,0x0e,0x77,0x84,0xa2,0x20,0x23,0x58,0xcd,0x31,0x3c};
```
Comment (profezzorn) [inline sound/voice_data.h:37]: So, these symbols only exists if you have FEMALE_TALKIE_VOICE defined?
I think that if there is no male version of the sound, it should go outside of the #ifdef.

File: sound/voice_data.h [hunk 3]
```diff
@@ -15,7 +15,6 @@
 // They also have good expression.
 // 
 #ifdef FEMALE_TALKIE_VOICE
-const uint8_t spBATTERYLVL[] PROGMEM = {0xa8,0xf3,0xc2,0xdd,0xc3,0x93,0xa8,0xb6,0x69,0x17,0x0b,0xb7,0xdd,0xac,0xac,0x35,0xaa,0x85,0xf6,0xb0,0xab,0xb1,0xa8,0x11,0xda,0xcb,0x29,0x36,0xa3,0x4a,0x65,0x2f,0xa7,0x84,0xf4,0x2a,0x75,0xb2,0x9c,0xea,0xd3,0xb3,0x4d,0xce,0x72,0xaa,0x0b,0x8b,0x76,0x39,0xc3,0x6d,0x2a,0xdc,0x32,0xa4,0x0c,0xb7,0xaa,0x30,0xcb,0x88,0xd2,0x9c,0xea,0x54,0x2a,0x22,0x6c,0x71,0x4b,0x42,0x1b,0xcb,0x30,0xc5,0x29,0x01,0xcb,0xb2,0x65,0x37,0xa7,0x07,0x4c,0x1d,0x93,0xdd,0xdc,0xe1,0xd0,0x63,0x5c,0x6e,0x72,0x0b,0xa7,0x9e,0x61,0xbb,0xce,0xd3,0xce,0xaa,0x57,0x2d,0xbb,0xc0,0xf2,0x6e,0xcb,0x88,0x14,0x02,0xaf,0xcc,0xd3,0x7c,0xb1,0x8b,0xac,0x6a,0xb1,0xf6,0xda,0x21,0xd4,0xbe,0x34,0xa2,0x43,0x97,0x20,0xfa,0xb2,0x1a,0xb4,0x3b,0xa2,0x6c,0xc7,0x63,0x45,0xc9,0x88,0xb3,0x9b,0xf0,0x51,0x36,0x23,0xce,0x31,0xbd,0x56,0x54,0xb7,0x24,0xfb,0x72,0x5f,0x63,0x62,0xd2,0x68,0xca,0xb9,0x9c,0xb6,0x8b,0x9a,0x08,0xd7,0x30,0x47,0x29,0x4c,0x7a,0x1d,0xbb,0x65,0xa7,0x30,0xaa,0x73,0x98,0x96,0x1c,0x82,0xa0,0xcf,0x60,0x8b,0xb6,0x0b,0x9c,0x5d,0xe5,0x8e,0x58,0x22,0xf6,0xa1,0x44,0x3a,0x23,0x03,0xc0,0x03};
```
Comment (profezzorn) [inline sound/voice_data.h:18]: Still removing this?

File: sound/voice_data.h (hunk 1 above)
Comment (NoSloppy) [reply sound/voice_data.h]: fixed.

File: sound/voice_data.h (hunk 2 above)
Comment (NoSloppy) [reply sound/voice_data.h:36]: fixed

File: sound/voice_data.h (hunk 3 above)
Comment (NoSloppy) [reply sound/voice_data.h:18]: fixed

File: sound/voice_data.h [hunk 4]
```diff
@@ -35,7 +33,10 @@ const uint8_t spGOOD[] PROGMEM = {0x0A,0x28,0xCD,0x34,0x20,0xD9,0x1A,0x45,0x74,0
...
+const uint8_t spSTUN[] PROGMEM = {0xe6,0x9f,0xa9,0xdd,0x2b,0xed,0x10,0x60,0x99,0x0e,0x02,0x4c,0xd3,0x41,0x80,0x69,0x3b,0x11,0xb0,0x5d,0x17,0x10,0x60,0xa9,0xcc,0xe6,0x36,0xe1,0xa6,0x1e,0xd6,0x96,0x97,0xad,0x85,0x8e,0x2a,0x3e,0x5e,0x0e,0x11,0xda,0x22,0x7b,0xf9,0x39,0xa5,0x6a,0x29,0xe3,0x11,0xe6,0x1c,0xe2,0x29,0xb2,0x57,0x5c,0x52,0xaa,0x96,0xc8,0x5e,0x49,0x0d,0xa9,0xda,0x2a,0xbb,0x65,0xd5,0xbb,0xca,0xb0,0xec,0x90,0x45,0x23,0x6a,0xad,0xaa,0x4d,0xae,0x3c,0x57,0x85,0x78,0x76,0x85,0x76,0x12,0x95,0xaa,0xc7,0x14,0xdc,0x6b,0x66,0xaa,0x6f,0x55,0xd0,0x60,0x15,0x25,0x9e,0x55,0xc1,0x74,0x64,0x74,0x70,0x25,0x45,0x34,0xaa,0xe6,0xc6,0x16,0x1e};
+#else // Male Voice
```
Comment (profezzorn) [inline sound/voice_data.h:39]: This is confusing, Move the comment after the #else

File: sound/voice_data.h (hunk 3 above)
Comment (profezzorn) [reply sound/voice_data.h:18]: Looks like spBATTERYLEVEL and spBATTLELEVELCURR are still being deleted?

Comment (NoSloppy) [reply sound/voice_data.h:18]: Yes, unless you prefer them in. 
I couldn't pull off a good sounding Male BATTLVLCURR, so there's only a female version. 
Neither that nor BATTERYLVL are not used anywhere, so I figure yes, delete them.

File: sound/voice_data.h (hunk 4 above)
Comment (NoSloppy) [reply sound/voice_data.h]: Moved.

File: sound/voice_data.h (hunk 3 above)
Comment (profezzorn) [reply sound/voice_data.h:18]: That's fine, but it looks weird when you do it in an unrelated PR, and the description of the PR doesn't mention it.

**Pull request merged**